### ISSUE TITLE
Error while using setup with model_deeplabv3_resnet50

### ISSUE THREAD

**Issue by cheshanf** [mlverse/luz#161]
Error occur while using setup function  with model_deeplabv3_resnet50 from torchvision package

library(luz)
library(torchvision)

model_v3 <- model_deeplabv3_resnet50(
  pretrained = FALSE,
  progress = TRUE,
  num_classes = OUT_CLASSES,
  pretrained_backbone = FALSE
)
model <- setup(model_v3, loss = nn_cross_entropy_loss(), optimizer = optim_adam)

Error in x$get_inherit() : attempt to apply non-function
